### PR TITLE
[Testing] Allow NCHW layout in `relay_workload`

### PR DESCRIPTION
**Pull request by junrushao**

### CODE REVIEW

**Comment by junrushao** [comment]
CC: @zxybazh

File: python/tvm/meta_schedule/testing/relay_workload.py [hunk 1]
```diff
@@ -86,27 +89,29 @@ def _get_network(
         input_name = "input0"
         shape_list = [(input_name, input_shape)]
         mod, params = relay.frontend.from_pytorch(scripted_model, shape_list)
+        passes = [relay.transform.RemoveUnusedFunctions()]
+        if layout == "NHWC":
```
Comment (zxybazh) [inline python/tvm/meta_schedule/testing/relay_workload.py:93]: If layout is `NCHW` do we need to also add layout transformation?

**Review by zxybazh — COMMENTED** [review]
Just one question to make sure I understand correctly.

**Review by zxybazh — APPROVED** [review]
Just find out the workloads are NCHW by default, so there's no problem only defining NHWC transformations.

File: python/tvm/meta_schedule/testing/relay_workload.py (hunk 1 above)
Comment (junrushao) [reply python/tvm/meta_schedule/testing/relay_workload.py:93]: added a comment saying pytorch is imported as nchw

**Pull request merged**